rasa/core/domain.py

```diff
@@ -124,7 +126,7 @@ def load(cls, filename):
     @classmethod
     def from_yaml(cls, yaml):
         cls.validate_domain_yaml(yaml)
-        data = read_yaml_string(yaml)
+        data = read_yaml(yaml)
```

Comment:
@tmbo We prefer it to have module name in front of imported functions, so that it's clear where it's coming from, right?